pennylane/templates/utils.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2018 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Utility functions used in the templates.
+"""
+#pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
+from collections.abc import Iterable
+import numpy as np
+from pennylane.qnode import Variable
+
+
+def _check_no_variable(arg, arg_str, msg=None):
```

Comment:
It would be nice to have a short docstring here

Reply:
Oh sorry